requirements/edx/base.txt

```diff
@@ -6,6 +6,7 @@
 
 beautifulsoup4==4.1.3
 beautifulsoup==3.2.1
+before_after==0.1.3
```

Comment:
@efischer19 Oh, actually can you please put it in the "# Used for testing" section instead. At least that way we'll know that it's only for testing and can hopefully untangle those later.